web/client/utils/WMTSUtils.js

```diff
@@ -20,7 +20,7 @@ const WMTSUtils = {
         return matrixIds;
     },
     getMatrixIds: (matrix, srs) => {
-        return (isObject(matrix) && matrix[srs] || matrix).map((el) => el.identifier);
+        return ((isObject(matrix) && matrix[srs]) || isArray(matrix) || []).map((el) => el.identifier);
```

Comment:
if `(isObject(matrix) && matrix[srs])` is true the `.map` iterates over an object.

Reply:
matrix is an object, matrix[srs] is an array

Reply:
You are right. ok